mk/setup_dpdk.py

```diff
@@ -1,1182 +0,0 @@
-#! /usr/bin/env python
```

Comment:
We need to keep this file somehow in the new build system, it's the script to configure DPDK after installing